src/include/migraphx/check_shapes.hpp

```diff
@@ -228,6 +228,13 @@ struct check_shapes
         return *this;
     }
 
+    const check_shapes& same_layout() const
```

Comment:
add docstring similar to other functions

Reply:
@pfultz2  can you please make these changes ?

Reply:
I already did.